### DISCUSSION TITLE
Hólmgöngur bannaðar - 2023/5/5

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#20] [vefsida-ummæli]
# Hólmgöngur bannaðar - 2023/5/5

Sagan af því hvernig hólmgöngur urðu bannaðar á Íslandi

https://www.biskupsreglan.com/verkefni/islendingasogur/holmgongur-bannadar

**Comment by TJ478** [comment]
Þetta er stórkostlegur atburður í sögu okkar á Íslandi.
